nested/controllerconfig.json

```diff
@@ -8,9 +8,9 @@
             },
             "type": "object"
         },
-        "blobStorageAccountKey": {
+        "storageAccountKey": {
             "metadata": {
-                "description": "Key used to create a storage container"
+                "description": "Key used to create a storage container and Azure Files share if azurefiles is specified as fileServerType"
```

Comment:
Should probably remove the reference to Azure Files in the description. Lets leave it as an undocumented feature given uncertainty of performance.

Reply:
Removed